### PR TITLE
Unittest/health api

### PR DESCRIPTION
**Pull request by extremeFunk**

### CODE REVIEW

File: awesome_provider/app/Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+WORKDIR /app
+RUN python3 -m pip install --no-cache-dir -r requierments.txt
+COPY . /app
+
+
+CMD  python3 -m unittest discover -s tests -p 'test_*.py'
```
Comment (asi91) [inline awesome_provider/app/Dockerfile.test]: ?

File: awesome_provider/app/src/routing.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from flask import Flask, request
+from model import Model
+import os
+import sys
```
Comment (asihamza) [inline awesome_provider/app/src/routing.py]: Change the order:
```
import os, sys
..
```

File: awesome_provider/app/src/routing.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+if __name__ == '__main__':
+    dbUrl = os.environ['DB_URL']
+    dbUser = os.environ['DB_USR']
+    dbPass = os.environ['DB_PASS']
+    dbName = os.environ['DB_NAME']
+    dbPort = int(os.environ['DB_PORT'])
+    qHelper = QueryHelper(dbUrl, dbUser, dbPass, dbName, dbPort)
+    model = Model(qHelper)
+
+    app.run(host='0.0.0.0', debug=True)
```
Comment (asihamza) [inline awesome_provider/app/src/routing.py]: Remove `debug`

File: awesome_provider/app/src/stam.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from foo import Foo
+
+class Stam:
+    def omer(self):
+        return Foo().bar()
```
Comment (asihamza) [inline awesome_provider/app/src/stam.py]: What is this?

File: awesome_provider/app/tests/test_app.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import unittest
```
Comment (asihamza) [inline awesome_provider/app/tests/test_app.py]: Redundant

File: awesome_provider/app/tests/test_app.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        #provider_routing.set_model(mock_model)
+        response = self.client.get('/health')
+        self.assertEqual(200, response.status_code)
+
+
+#python3 -m venv virt
+#source virt/bin/activate
+#sudo apt install default-libmysqlclient-dev
+#pip3 install -r requierments.txt
+#python -m unittest discover ../tests/
```
Comment (asihamza) [inline awesome_provider/app/tests/test_app.py]: Remove commented code

File: awesome_provider/docker-compose-test.yml [hunk 7]
```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+
+    services:
```
Comment (asihamza) [inline awesome_provider/docker-compose-test.yml]: Redundant `services`?

**Review by asihamza — CHANGES_REQUESTED** [review]

**Review by asihamza — APPROVED** [review]

**Pull request merged**